WolvenKit.Common/Model/Arguments/ExportArgs.cs

```diff
@@ -194,13 +194,13 @@ public class MeshExportArgs : ExportArgs
         private bool _isGLBinary = true;
 
         /// <summary>
-        /// Experimental New Mesh Exporter Logic.
+        /// Choose exporter
         /// </summary>
         [Category("Experimental")]
-        [Display(Name = "Use New Mesh Exporter")]
-        [Description("Use New Mesh Exporter Logic. Turn off if you run into issues.")]
+        [Display(Name = "Choose Exporter")]
+        [Description("Choose the exporter type. Use default if you run into issues.")]
         [WkitScriptAccess("ExperimentalNewMeshExporter")]
-        public bool ExperimentUseNewMeshExporter { get; set; } = false; // TODO: Remove this when the new exporter is stable.
+        public MeshExporterType MeshExporter { get; set; }
```

Comment:
Maybe change category to `General` or something like that?
Also please change the ScriptAttribute (or delete the name param from it)

Reply:
oh right sorry I forgot